File: apzkr-pzpi-21-11-milennyi-ivan/Task3-WebClient/src/viewModels/StorekeeperVM.ts
// Клас для відображення інформації про комірника
export class StorekeeperVM {
    id: number; // Ідентифікатор комірника
    name: string; // Ім'я комірника
    surname: string; // Прізвище комірника

    constructor(id: number, name: string, surname: string) {
        this.id = id;
        this.name = name;
        this.surname = surname;
    }
}

File: apzkr-pzpi-21-11-milennyi-ivan/Task1-Server/src/view_models/storekeeper_vm.rs
use serde::Serialize;
use sqlx::FromRow;
use utoipa::ToSchema;

// Структура, що представляє представлення комірника (ViewModel)
#[derive(Debug, Serialize, FromRow, ToSchema)]
pub(crate) struct StorekeeperVM {
    id: u64, // Ідентифікатор комірника
    name: String, // Ім'я комірника
    surname: String, // Прізвище комірника
}
